src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
I found it, you are right I create this function. It is override to template function for special case when it receive refract element. 

But I still don't understand what do you mean by:
> Please use that by adding a parameter.